apps/microtvm/reference-vm/base-box-tool.py

```diff
@@ -173,12 +173,19 @@ def attach_vmware(uuid, vid_hex=None, pid_hex=None, serial=None):
     "vmware_desktop": attach_vmware,
 }
 
+EXTRA_SCRIPTS = (
+    "../../../../../docker/install/ubuntu_init_zephyr_project.sh",
```

Comment:
can you fetch repo root with
`subprocess.check_output(["git", "rev-parse", "--show-toplevel"], cwd=os.path.dirname(__file__))`

and then express these paths relative to that?